### DISCUSSION TITLE
Runelite pmods, global mute list, crowdsourced spam filters

### DISCUSSION THREAD

**Discussion by tuckershannon** [runelite/runelite#16590] [Ideas]
Crowdsourced mute and spam filter list

Right click, extra option, mute player. Text saved from player and muted player ign added to global list. Text analyzed, determines if it's spam based on sample data from global mute logs. Players will show a "." If their message is muted meaning it didn't pass spam filter. Accounts global muted. 

Also possible, replace in game p mods with runelite "pmods". Mutes made by runelite pmods would global mute those accounts for some amount of time

**Comment by Felanbird** [comment]
This sounds like a huge project for very little gain

**Reply by tuckershannon** [reply]
I think Jagex thinks the same thing about their mute system. Which may be why the ge looks the way it does with spam bots everywhere.
